server/prisma/generator/factories/events.factory.ts

```diff
@@ -84,19 +84,46 @@ const createEvents = async (
       }),
     );
 
-    const tagNames = Array.from(new Array(1 + random(3)), () => lorem.word());
+    const tagNames = [
+      'GraphQl',
+      'NodeJs',
+      'JavaScript',
+      'TypeScript',
+      'HTML',
+      'CSS',
+      'Cypress',
+      'Tailwind',
+      'Sass',
+      'BootStrap',
+      'React',
+      'Vue',
+      'NextJs',
+      'NuxtJs',
+      'Angular',
+      'Svelete',
+      'SveleteKit',
+      'Vite',
+      'Prisma',
+      'Ruby',
+      'Rust',
+    ];
+    const tagsAmount = Math.round((Math.random() * tagNames.length) / 5);
+
+    const tags = randomItems(tagNames, tagsAmount, true);
+    const connectOrCreateTags = tags.map((tag) => ({
+      tag: {
+        connectOrCreate: {
+          where: { name: tag },
+          create: { name: tag },
+        },
+      },
+    }));
```

Comment:
```suggestion
    const tagNames = randomItems(tagNames, tagsAmount, true);
    const connectOrCreateTags = tagNames.map((name) => ({
      tag: {
        connectOrCreate: {
          where: { name },
          create: { name },
        },
      },
    }));
```
I know I suggested the first way of naming it, but this seems nicer!